src/interfaces/modular/SGBase.i

```diff
@@ -107,6 +108,7 @@ public void readExternal(java.io.ObjectInput in) throws java.io.IOException, jav
  #include <shogun/base/Version.h>
  #include <shogun/base/Parallel.h>
  #include <shogun/base/SGObject.h>
+ #include <../tests/unit/base/MockObject.h>
```

Comment:
for relative includes please use `""`, but why do we include `MockObject.h` from (unit) tests into the code itself?